lib/src/controls/navigation/tree_view.dart

```diff
@@ -309,6 +309,14 @@ extension TreeViewItemCollection on List<TreeViewItem> {
       child.children.executeForAll(callback);
     }
   }
+
+  Iterable<TreeViewItem> whereForAll(bool Function(TreeViewItem element) t) {
```

Comment:
could be private

Reply:
my preference would be to keep it public. similar to how `executeForAll` is public, it's handy to have `whereForAll` be public as well. It's useful for my application code to easily filter the list of all items in the tree for some condition (e.g., doing a search over items).